wwwroot/inc/snmp.php

```diff
@@ -4361,6 +4370,30 @@
 		'text' => 'HP 1810-8G (J9449A), 8 Gigabit Ethernet 10/100/1000',
 		'processors' => array ('procurve-8ports-1000T'),
 	),
+	'9.1.1643' => array
+	(
+		'dict_key' => 3728,
+		'text' => 'Cisco Catalyst 3850-48T-S Switch, 48 10/100/1000',
+		'processors' => array ('cisco-smb-49-to-50-combo-1000SFP','cisco-smb-any-1000T'),
+	),
+	'9.1.571' => array
+	(
+		'dict_key' => 3731,
+		'text' => 'Cisco 871',
+		'processors' => array ('catalyst-chassis-any-100TX'),
+	),
+	'9.12.3.1.3.1239' => array
+	(
+		'dict_key' => 2332,
+		'text' => 'Cisco Nexus 3064-T Switch,  48 x 10GBase-T + 4 x QSFP+',
+		'processors' => array ('nexus-any-QSFP+','nexus-any-10000SFP+'),
```

Comment:
'nexus-any-QSFP-split' (existing) and 'nexus-any-10000T' (needs to be added similar to 'nexus-any-10000SFP+'